.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+    push:
+        branches:
+            - master
+jobs:
+    release:
+        name: Build and Release
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+            issues: write
+            pull-requests: write
```

Comment:
Do we need all of these permissions? 
AFAIK we just need contents permission 🤔

Reply:
We need permissions for pull requests and issues to be able to write [these](https://github.com/deriv-com/deriv-utils/pull/38#issuecomment-2044675729) comments